contracts/token/redeemable/token/RCSTKToken.sol

```diff
@@ -108,43 +107,38 @@ contract RCSTKToken is
         uint256 softCapTimestamp; /// @dev when the softcap was reached
         uint256 totalReceived; /// @dev total DAI received in this iteration
     }
-    /// TODO!! Make currentState readable
+
     /// @notice State of the fundraise
     enum State {CREATED, ACTIVE, INACTIVE, END_RAISE_STARTED, FINISHED}
     State internal currentState;
 
-    /// TODO!! Make readable
     /// @notice Number of existing iterations
     uint256 internal numIterations;
 
-
     /// @notice List of iterations: iterations[index]
-    mapping(uint256 => Iteration) internal iterations;
+    mapping(uint256 => Iteration) internal _iterations;
```

Comment:
Question: do we prefix storage vars with `_`? Or just function parameters?

Reply:
I prefixed internal variables, open to suggestions on consistent rules.